webpack/components/ResourceQuotaEmptyState/index.js

```diff
@@ -28,16 +29,32 @@ const ResourceQuotaEmptyState = () => {
       {__('Create resource quota')}
     </Button>
   );
+
+  const description = (
+    <span>
+      {__(
+        'Resource Quotas help admins to manage hardware resources including CPUs, memory, and disk space among users or usergroups.'
+      )}
+      <br />
+      {__(
+        'Define a Resource Quota here and apply it to users in order to guarantee a free share of your resources.'
```

Comment:
question: Do you mean "fair share"?

suggestion: drop "in order"

Reply:
Implemented both, thanks!